ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/ClusterStatusResource.java

```diff
@@ -102,7 +102,7 @@ private HostStoreLags getHostStoreLags(final KsqlHostInfo ksqlHostInfo, final bo
     final Map<String, ActiveStandbyEntity> perQueryMap = new HashMap<>();
     for (PersistentQueryMetadata persistentQueryMetadata: engine.getPersistentQueries()) {
       for (StreamsMetadata streamsMetadata: persistentQueryMetadata.getAllMetadata()) {
-        if (streamsMetadata.hostInfo().equals(asHostInfo(ksqlHostInfo))) {
+        if (!streamsMetadata.hostInfo().equals(asHostInfo(ksqlHostInfo))) {
```

Comment:
https://github.com/confluentinc/ksql/commit/9615a0296bfacd6eea9f054958319987792865f4#diff-e7ce1cd863f76d6f652a7345d1289367dbf96a963cdd63afe2c192c1c0c5a944L106-L107 dropped the negation in front of `streamsMetadata.hostInfo().equals(asHostInfo(ksqlHostInfo)`.